MicrosoftFluentUI.podspec

```diff
@@ -281,9 +283,20 @@ Pod::Spec.new do |s|
     core_mac.source_files = ["macos/FluentUI/Core/**/*.{swift,h}"]
   end
 
+  s.subspec 'AppearanceExtension_mac' do |appearanceextension_mac|
+    appearanceextension_mac.platform = :osx
+    appearanceextension_mac.source_files = ["macos/FluentUI/AppearanceExtensions/**/*.{swift,h}"]
```

Comment:
Let's make the folder name just **Appearance**.
Same for the subspec name: **Appearance_mac**